### PR TITLE
Update misk publishing (wow so many gradle.properties are gone)

### PR DESCRIPTION
**Pull request by tso**

### CODE REVIEW

File: .github/workflows/Release.yml [hunk 1]
```diff
@@ -26,9 +26,15 @@ jobs:
           sed --in-place -e "s/-SNAPSHOT/-$(date +%Y%m%d.%H%M)-$(git rev-parse --short HEAD)/" gradle.properties
           cat gradle.properties
 
+      - name: Configure JDK
```
Comment (tso) [inline .github/workflows/Release.yml:29]: copied from tempest, do we want this?

File: build.gradle.kts [hunk 2]
```diff
@@ -188,3 +189,35 @@ subprojects {
     }
   }
 }
+
+allprojects {
```
Comment (tso) [inline build.gradle.kts:193]: should this be combined with the above `allprojects` block?

Comment (swankjesse) [reply build.gradle.kts:193]: I believe that having lots of independent `allProjects` blocks is best for keeping logic self-contained

**Review by swankjesse — APPROVED** [review]

File: .github/workflows/Release.yml (hunk 1 above)
Comment (swankjesse) [reply .github/workflows/Release.yml]: This should track whatever the Build.yml does

**Pull request merged**